source/Cute/Commands/Content/ContentTranslateCommand.cs

```diff
@@ -211,37 +230,74 @@ await ProgressBars.Instance()
                                     {
                                         tService = TranslationService.GPT4o;
                                     }
+
                                     var translator = _translateFactory.Create(tService);
-                                    try
+
+                                    Task<(string?, bool)> translateWrapper() => Task.Run(async () =>
```

Comment:
The translateWrapper() function creates a new task inside a task, which adds overhead
The Task.Run() could be eliminated by making the lambda async directly